code/modules/procedural_mapping/mapGenerators/lavaland.dm

```diff
@@ -10,15 +10,15 @@
 
 /datum/mapGeneratorModule/splatterLayer/lavalandMonsters
 	spawnableTurfs = list()
-	spawnableAtoms = list(/mob/living/simple_animal/hostile/asteroid/goliath/beast = 10,
-	/mob/living/simple_animal/hostile/asteroid/hivelord/legion = 10,
-	/mob/living/simple_animal/hostile/asteroid/basilisk/watcher = 10)
+	spawnableAtoms = list(/mob/living/simple_animal/hostile/asteroid/goliath/beast = 20,
+	/mob/living/simple_animal/hostile/asteroid/hivelord/legion = 20,
+	/mob/living/simple_animal/hostile/asteroid/basilisk/watcher = 20)
 
 /datum/mapGeneratorModule/splatterLayer/lavalandTendrils
 	spawnableTurfs = list()
-	spawnableAtoms = list(/mob/living/simple_animal/hostile/spawner/lavaland = 5,
-	/mob/living/simple_animal/hostile/spawner/lavaland/legion = 5,
-	/mob/living/simple_animal/hostile/spawner/lavaland/goliath = 5)
+	spawnableAtoms = list(/mob/living/simple_animal/hostile/spawner/lavaland = 15,
+	/mob/living/simple_animal/hostile/spawner/lavaland/legion = 15,
+	/mob/living/simple_animal/hostile/spawner/lavaland/goliath = 15)

```

Comment:
Why are you tripling the amount of tendrils, legions, and goliaths on lava land?